arviz/stats/stats.py

```diff
@@ -305,7 +305,7 @@ def _ic_matrix(ics, ic_i):
     return rows, cols, ic_i_val
 
 
-def hpd(ary, credible_interval=None, circular=False, multimodal=False, skipna=False):
+def hpd(ary, *, credible_interval=None, circular=False, multimodal=False, skipna=False, **kwargs):
```

Comment:
we can also add a group argument (like the one in plotting functions) in case someone wanted to calculate hpd on posterior predictive, predictions...

also a `var_names` could be added.

Reply:
group argument is valid for InferenceData object input, right? Or is there some other case.